charts/azuremonitor-containers/README.md

```diff
@@ -85,7 +85,7 @@ The following table lists the configurable parameters of the MSOMS chart and the
 | -----------------------      | --------------------------------------------------------| --------------------------------------------------------------------------------------------------------------------------- |
 | `amalogs.image.tag`         | image tag for Linux Agent.                              | Most recent release                                                                                                         |
 | `amalogs.image.tagWindows`  | image tag for Windows Agent.                            | Most recent release                                                                                                         |
-| `amalogs.image.imagerepo`   | image repo for Liunx & Windows.                         | For Public and US Govt cloud: mcr.microsoft.com/azuremonitor/containerinsights/ciprod and For China Cloud: mcr.azk8s.cn/azuremonitor/containerinsights/ciprod
+| `amalogs.image.imagerepo`   | image repo for Liunx & Windows.                         | For Public, US Govt cloud and China Cloud: mcr.microsoft.com/azuremonitor/containerinsights/ciprod, and China Cloud will be redirected to mcr.azk8s.cn
```

Comment:
This will be mcr.microsoft.com/azuremonitor/containerinsights/ciprod for all the clouds and there is no direction hence update the from


From
For Public, US Govt cloud and China Cloud: mcr.microsoft.com/azuremonitor/containerinsights/ciprod, and China Cloud will be redirected to mcr.azk8s.cn
To 
Image repo path is mcr.microsoft.com/azuremonitor/containerinsights/ciprod

Reply:
changed